src/app/zap-templates/zcl/data-model/chip/operational-state-cluster.xml

```diff
@@ -35,15 +35,15 @@ limitations under the License.
 
   <struct name="OperationalStateStruct">
     <cluster code="0x0060"/>
-    <item name="OperationalStateID"    type="OperationalStateEnum" optional="false"/>
-    <item name="OperationalStateLabel" type="CHAR_STRING" optional="false" length="64"/>
+    <item name="OperationalStateID"    type="ENUM8" optional="false"/>
```

Comment:
But now this is not matching the spec, right?  Spec has OperationalStateEnum here.

Reply:
Indeed. Hence the PR below :)

https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7193